doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -17,120 +18,294 @@ Check out CITE suite tools
+
+   - Run ``make`` to check:
 
    .. code:: shell
 
-     ant wfs-1.0
-     
-   With the following parameters:
+        make
 
-   #. ``Capabilities URL`` http://localhost:8080/geoserver/wfs?request=getcapabilities&service=wfs&version=1.0.0
 
-   #. ``All`` tests included
+   - you should get an output as following:
 
-      .. image:: tewfs-1_0.png
+   .. code:: makefile
 
-Run WFS 1.1 tests
+        clean: $(suite)		 Will Clean the Environment of previous runs.
+        build: $(suite)		 Will Build the GeoServer Docker Image for the Environment.
+        test: $(suite)		 Will running the Suite test with teamengine.
+        webUI: $(suite)		 Will running the Suite test with teamengine.
+
+
+Run WFS 1.0 tests
 -----------------
 
 .. important::
 
-   Running WFS 1.1 tests require PostGIS to be installed on the system.
+   Running WFS 1.0 tests require PostgreSQL with PostGIS extension installed to be installed on the system.
 
 Requirements:
 ~~~~~~~~~~~~~
+- `GeoServer <https://github.com/randomorder/geoserver>_`
+- teamengine
 - Posgresql
 - PostGIS
 
+#. Start the test:
+
+   .. code:: shell
+
+     make webUI
 
-#. Create a PostGIS user named "cite":
+#. Go to the browser and open the URL: http://localhost:8888/teamengine/
 
-   .. code:: SQL
+   - after the site open, you need to create an account to run the tests.
 
-     createuser cite
+   With the following parameters:
 
-#. Create a PostGIS databased named "cite", owned by the "cite" user:
+   #. ``Capabilities URL`` http://<ip.of.the.goserver>:8080/geoserver/wfs?request=getcapabilities&service=wfs&version=1.0.0
 
-   .. code:: SQL
+   #. ``Enable tests with multiple namespaces`` tests included
 
-     createdb -T template_postgis -U cite cite
+      .. image:: ./image/tewfs-1_0.png
 
-#. Change directory to the ``citewfs-1.1`` data directory and execute the script ``dataset-sf0.sql``:
+Run WFS 1.1 tests
+-----------------
 
-   .. code:: SQL
+.. important::
 
-     psql -U cite cite < dataset-sf0.sql
+   Running WFS 1.1 tests require PostGIS to be installed on the system.
 
-#. Start GeoServer with the ``citewfs-1.1`` data directory. Example:
+Requirements:
+~~~~~~~~~~~~~
+- GeoServer
+- teamengine
+- Posgresql
+- PostGIS
 
-   .. code:: SHELL
 
-     cd <root of geoserver install>
-     export GEOSERVER_DATA_DIR=<root of geoserver sources>/build/cite/wfs11/citewfs-1.1
-     ./bin/startup.sh
+#. Start the test:
 
-#. Change directory back to the cite tools and run the tests:
+   .. code:: shell
+
+     make webUI
 
-   .. code:: SHELL
+#. Go to the browser and open the URL: http://localhost:8888/teamengine/
 
-     ant wfs-1.1
+   - after the site open, you need to create an account to run the tests.
 
    With the following parameters:
 
-   #. ``Capabilities URL`` http://localhost:8080/geoserver/wfs?service=wfs&request=getcapabilities&version=1.1.0
+   #. ``Capabilities URL`` http://<ip.of.the.goserver>:8080/geoserver/wfs?service=wfs&request=getcapabilities&version=1.1.0
```

Comment:
This is not enough, the GeoServer must be using a given data dir and setup the database with the right test data.

For example, see this one:
https://github.com/geoserver/geoserver/tree/main/data/citewfs-1.0

It had instructions on how to run the setup.  Please add this information for each test suite, so that the dev charged to fix GeoServer to pass a given suite, knows how to setup their computer.

Reply:
this topic was resolved in this commit https://github.com/randomorder/geoserver/commit/3f79a0684747816e281346ea4353d28e1930f565